apps/web/package.json

```diff
@@ -13,7 +13,7 @@
     "dx": "yarn dev",
     "test-codegen": "yarn playwright codegen http://localhost:3000",
     "type-check": "tsc --pretty --noEmit",
-    "type-check:ci": "tsc-absolute --pretty --noEmit",
+    "type-check:ci": "tsc --pretty --noEmit --skipLibCheck",
```

Comment:
*casually changes the entire workflow of the team, for worse* well, the builds will be easier for sure